server/templates/NewChart.template.js

```diff
@@ -1,15 +1,21 @@
 import basePathConfig from '../configs/basePathConfig'
+const serialize = require('serialize-javascript');
```

Comment:
should this use the `import` syntax? Also, I believe node module imports should come before local imports with a newline separating them.